packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadiness.ts

```diff
@@ -161,13 +162,16 @@ export function useTransferReadiness({
       return null
     }
 
+    const { isOrbitChain } = isNetwork(childChain.id)
+
     const isL2NativeUSDC =
       isTokenArbitrumOneNativeUSDC(selectedToken.address) ||
       isTokenArbitrumSepoliaNativeUSDC(selectedToken.address)
 
-    const selectedTokenL2Address = isL2NativeUSDC
-      ? selectedToken.address.toLowerCase()
-      : (selectedToken.l2Address || '').toLowerCase()
+    const selectedTokenL2Address =
+      isL2NativeUSDC && !isOrbitChain
```

Comment:
Without the check for `!isOrbitChain` it would read parent chain address when Orbit chain is selected. That's why it read balance as `null` and blocked the withdrawal.